Sources/SnowManLogic/ViewLogic/MockView.swift

```diff
@@ -0,0 +1,45 @@
+public class MockView: Viewable {
```

Comment:
Good question. I was using it previously in my gameSpec. I ended up making the method that called the view private so I do not need this anymore. I guess that would also make the protocol unnecessary.